internal/git_credentials_test.go

```diff
@@ -29,6 +22,10 @@ func TestAccGitCredentials(t *testing.T) {
 		require.NoError(t, err)
 	})
 
+	list, err := w.GitCredentials.ListAll(ctx)
+	require.NoError(t, err)
+	assert.True(t, len(list) >= 1)
```

Comment:
nit: instead of this, would it be better to check for the git provider and username we set above?